code/modules/mob/living/carbon/human/species_types/synths.dm

```diff
@@ -33,9 +33,8 @@
 	if(chem.id == "synthflesh")
 		chem.reaction_mob(H, TOUCH, 2 ,0) //heal a little
 		H.reagents.remove_reagent(chem.id, REAGENTS_METABOLISM)
-		return 1
-	else
-		return ..()
+	..()

```

Comment:
Doesn't this defeat the purpose?

Reply:
this calls
```/datum/species/proc/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
	if(chem.id == exotic_blood)
		H.blood_volume = min(H.blood_volume + round(chem.volume, 0.1), BLOOD_VOLUME_MAXIMUM)
		H.reagents.del_reagent(chem.id)
		return 1
return FALSE```
which I guess doesn't matter